modules/retrieving-debugging-information.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * cli/debugging-issues.adoc
+:_content-type: PROCEDURE
+[id="retrieving-debugging-information_{context}"]
+= Retrieving debugging information
+
+.Procedure
+
+* Run the following command to gather the debugging information for investigating issues:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" central debug dump
+----
```

Comment:
We should probably also add the diagnostic bundle command here, which is preferred in support cases over the debug dump (the debug dump exposes a bit more information about mem, CPU usage etc. so the diagnostic bundle is the thing to start with in most cases).